mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java

```diff
@@ -94,6 +94,9 @@
     @CommandLine.Option(names = "--hsm-metadata-signing-label", description = "HSM Metadata key label")
     private String hsmMetadataKeyLabel = "private_key";
 
+    @CommandLine.Option(names = "--saml-encryption-cert", description = "Public X509 cert for metadata encryption if required")
```

Comment:
This doesn't quite match the work we've done on mdgen in 1548, but it'll do for now.